### PR TITLE
User auth

### PR DESCRIPTION
**Pull request by LuisRococo**
Add user model and devise auth with email confirmation.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ ruby '3.0.4'
 gem 'rails', '~> 6.1.7'
 # Use sqlite3 as the database for Active Record
 gem 'sqlite3', '~> 1.4'
+gem 'devise'
```
Comment (ohenroute) [inline Gemfile:10]: is devise in the database group? why not use the command to use Postgres as well... or change the Gem.

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class PagesController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:home]
```
Comment (ohenroute) [inline app/controllers/pages_controller.rb:2]: filters! 👍

File: config/environments/development.rb [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (ohenroute) [inline config/environments/development.rb:6]: Aren't we using letter opener?

File: config/locales/devise.en.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/heartcombo/devise/wiki/I18n
```
Comment (ohenroute) [inline config/locales/devise.en.yml:1]: Remember to keep adding to the translations.

**Review by ohenroute — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (LuisRococo) [reply Gemfile:10]: Devise is not inside of any group at the time. I am going to configurate postgres in a moment.

File: config/environments/development.rb (hunk 3 above)
Comment (LuisRococo) [reply config/environments/development.rb:6]: I wanted to test a little bit with the default configuration. I am going to add the gem as well in a moment.

File: config/locales/devise.en.yml (hunk 4 above)
Comment (LuisRococo) [reply config/locales/devise.en.yml:1]: Ok, I will have them in mind.

File: Gemfile (hunk 1 above)
Comment (ohenroute) [reply Gemfile:10]: Understood. I will wait up for the next PR. 

Good job! 👍

File: config/environments/development.rb (hunk 3 above)
Comment (ohenroute) [reply config/environments/development.rb:6]: Understood. 👍 

Good job.

**Comment by ohenroute** [comment]
Keep on the good effort 👍

**Review by ohenroute — APPROVED** [review]

**Pull request merged**